src/i18n/strings/cs.json

```diff
@@ -3168,7 +3168,7 @@
     "Open": "Otevřít",
     "Share decryption keys for room history when inviting users": "Při pozvání uživatelů sdílet dešifrovací klíče pro historii místnosti",
     "Manage & explore rooms": "Spravovat a prozkoumat místnosti",
-    "Message search initilisation failed": "Inicializace vyhledávání zpráv se nezdařila",
+    "Message search initialisation failed": "Inicializace vyhledávání zpráv se nezdařila",
```

Comment:
changing strings in the translations ends up causing conflicts with weblate - we only commit the output of `yarn i18n` and leave the rest to weblate to handle

Reply:
Yeah but doesn't that end up meaning the translators have to retranslate it? Or is weblate smart enough to figure out the source string changed but only a little bit so it's probably the same word.

Reply:
Yes, they'd have to re-translate it.

Reply:
Unfortunately, Weblate is annoying enough to manage that we tend to avoid conflicts at all costs. Yes, it means translations are lost and need to be re-done.

Reply:
Alright I will manually put the old translations in to Weblate then once this gets merged